Ira/movies-app/src/components/homePage.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import Link from "react-router-dom";
+import Header from './header';
+import moviesData from './moviesData';
+import MovieCard from './movieCard'; 
+import NotFound from './notFound'; 
+import Layout from './layout';
+
+
+
+interface IHomePageState {
+   filteredMovies: Array<{}>;
+   searchInputValue: string;
+   searchBy: string;   
+   moviesCount: number;
+}
+
+class HomePage extends React.Component<IHomePageState> {   
+   
+   state = {
+      filteredMovies: [{}],   
+      searchInputValue: "",
+      searchBy: "title",
```

Comment:
```
searchInputValue: "",
searchBy: "title" 
```
These fields better to move to the SearchFormComponent and manage them inside of it, so it will be easier to reuse in future; you won't need to create state future in every component where you need searchForm.